### PR TITLE
Moving the multi-Account changes to feature Extension

### PR DESCRIPTION
**Pull request by Johnbush**
**Description of the change**

- Move the trigger Jenkins job file to feature-extensions.
- Removed copy scripts from all scenerios
- Validated and fixed flake 8 issues.

### CODE REVIEW

File: feature-extensions/multiaccount/triggerJenkinsDeleteResources.py [hunk 1]
```diff
@@ -35,6 +37,10 @@ def startJob(args):
             args.jenkins_password, args.account_details)
 
     subprocess.call(jenkins_build_command, shell=True)
+    if jenkins_job_status("delete-resources"):
```
Comment (rajeevr2715) [inline feature-extensions/multiaccount/triggerJenkinsDeleteResources.py]: Please change the messages, if we dont need we can remove them

File: feature-extensions/multiaccount/triggerJenkinsDeleteResources.py [hunk 2]
```diff
@@ -56,4 +62,23 @@ def collectUserInput(args):
...
 
+def jenkins_job_status(job_name, args):
+    try:
+        url = "http://%s:%s@%s/job/%s/lastBuild/api/json" \
+                % (args.jenkins_username, args.jenkins_password, args.jenkins_url, job_name)
+        data = requests.get(url).json()
+        if data['building']:
+            time.sleep(60)
+        else:
+            if data['result'] == "SUCCESS":
```
Comment (rajeevr2715) [inline feature-extensions/multiaccount/triggerJenkinsDeleteResources.py]: Please change the messages, if we dont need we can remove them

Comment (Johnbush) [reply feature-extensions/multiaccount/triggerJenkinsDeleteResources.py]: Removed.

**Comment by rajeevr2715** [comment]
LGTM

**Pull request merged**